pkg/ingress/kube/common/tool.go

```diff
@@ -193,7 +196,27 @@ func SortHTTPRoutes(routes []*WrapperHTTPRoute) {
 		}
 
 		if routes[i].OriginPathType == routes[j].OriginPathType {
-			return len(routes[i].OriginPath) > len(routes[j].OriginPath)
+			if n, m := len(routes[i].OriginPath), len(routes[j].OriginPath); n != m {
```

Comment:
I do not prefer the names `n` and `m`, they are lack of readability.

Reply:
I see. how about `len1`, `len2` or `leni`, `lenj` or `pathLen1`, `pathLen2`?